package.json

```diff
@@ -26,6 +26,8 @@
   },
   "dependencies": {
     "chalk": "^1.1.3",
-    "minimist": "^1.2.0"
+    "htmltojsx": "git://github.com/harrison/react-magic.git",
```

Comment:
Good point. React-magic doesn't appear to be on npm - though we can mark it at a certain commit hash. So the package file would/should look like: `"htmltojsx": "git://github.com/harrison/react-magic.git#3ab31b8"`

HTMLtoJSX is required on line 10 of index.js and used on line 40. If viewing within the github site, the index.js diff is collapsed - so the changes/usage might not be apparent straight away.